.gitlab-ci.yml

```diff
@@ -1,8 +1,10 @@
 # TODO: GitLab's documentation recommends specifying a version:
```

Comment:
This TODO should be moved down to line 7.

Reply:
Done, thanks Leon!